README.rst

```diff
@@ -593,8 +601,9 @@ you to run your server on a machine that might not have the same name as your
 domain name. For example, you might want to run your server at
 ``synapse.example.com``, but have your Matrix user-ids look like
 ``@user:example.com``. (A SRV record also allows you to change the port from
-the default 8448. However, if you are thinking of using a reverse-proxy, be
-sure to read `Reverse-proxying the federation port`_ first.)
+the default 8448. However, if you are thinking of using a reverse-proxy on the
+federation port, which is highly not recommended, be sure to read
```

Comment:
s/highly//. If nothing else, you'll need a reverse-proxy if you have a synapse with split workers, so this is a bit strong.